src/main/java/com/zaxxer/hikari/metrics/micrometer/MicrometerMetricsTracker.java

```diff
@@ -2,33 +2,32 @@
 
 import com.zaxxer.hikari.metrics.IMetricsTracker;
 import com.zaxxer.hikari.metrics.PoolStats;
-import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.Counter;
 import io.micrometer.core.instrument.Gauge;
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.Timer;
+import io.micrometer.core.instrument.stats.hist.Histogram;
 import io.micrometer.core.instrument.stats.quantile.WindowSketchQuantiles;
 
 import java.util.concurrent.TimeUnit;
 
-import static io.micrometer.core.instrument.stats.hist.CumulativeHistogram.buckets;
-import static io.micrometer.core.instrument.stats.hist.CumulativeHistogram.linear;
-
 public class MicrometerMetricsTracker implements IMetricsTracker
 {
    private static final String METRIC_CATEGORY = "pool";
-   private static final String METRIC_NAME_WAIT = "Wait";
-   private static final String METRIC_NAME_USAGE = "Usage";
-   private static final String METRIC_NAME_CONNECT = "ConnectionCreation";
-   private static final String METRIC_NAME_TIMEOUT_RATE = "ConnectionTimeoutRate";
-   private static final String METRIC_NAME_TOTAL_CONNECTIONS = "TotalConnections";
-   private static final String METRIC_NAME_IDLE_CONNECTIONS = "IdleConnections";
-   private static final String METRIC_NAME_ACTIVE_CONNECTIONS = "ActiveConnections";
-   private static final String METRIC_NAME_PENDING_CONNECTIONS = "PendingConnections";
+   private static final String METRIC_NAME_WAIT = "hikaricp.connections.acquired.duration";
```

Comment:
"hikaricp.connections.acquired.duration" → "hikaricp.connections.acquire.duration"

Reply:
Drop the word "duration", as it is added by the Prometheus naming convention. It is not a suffix that is common in other monitoring systems.